cmake/llext-edk.cmake

```diff
@@ -158,13 +161,25 @@ yaml_get(INTERFACE_INCLUDE_DIRECTORIES NAME build_info KEY cmake llext-edk inclu
 yaml_get(APPLICATION_SOURCE_DIR NAME build_info KEY cmake application source-dir)
 yaml_get(WEST_TOPDIR NAME build_info KEY west topdir)
 
+yaml_get(board_name NAME build_info KEY cmake board name)
+yaml_get(board_qualifiers NAME build_info KEY cmake board qualifiers)
+yaml_get(board_revision NAME build_info KEY cmake board revision)
+
+set(board_target ${board_name})
+if(board_qualifiers)
+    set(board_target "${board_target}_${board_qualifiers}")
+endif()
+if(board_revision)
+    set(board_target "${board_target}_${board_revision}")
+endif()
+zephyr_string(SANITIZE board_target ${board_target})
```

Comment:
it seems to me that `zephyr_build_string()` is a better choice so that any changes or extensions to board terminology is automatically picked up. (SiP could be a future example).

https://github.com/zephyrproject-rtos/zephyr/blob/895d91169e8b6e573058a8a3f9fdf081f1362c81/cmake/modules/extensions.cmake#L1590-L1657